src/hooks/useCurrentBlockTimestamp.ts

```diff
@@ -1,10 +1,19 @@
 import { BigNumber } from '@ethersproject/bignumber'
+import { useEffect, useState } from 'react'
 
 import { useSingleCallResult } from '../state/multicall/hooks'
 import { useMulticall2Contract } from './useContract'
 
 // gets the current timestamp from the blockchain
 export default function useCurrentBlockTimestamp(): BigNumber | undefined {
+  const [lastBlockTimestamp, setLastBlockTimestamp] = useState<BigNumber | undefined>(undefined)
   const multicall = useMulticall2Contract()
-  return useSingleCallResult(multicall, 'getCurrentBlockTimestamp')?.result?.[0]
+  // TODO:zzmp
```

Comment:
what's the todo here? (going by commits, may be missing where you remove this)

Reply:
I was just marking the file with a diff so I could find it again.